packages/api/src/pins.js

```diff
@@ -355,6 +356,16 @@ function parseSearchParams (params) {
     opts.limit = limit
   }
 
+  if (meta) {
+    if (!JSON.parse(meta) || Array.isArray(meta) || Object.entries(meta).some(([, v]) => typeof v !== 'string')) {
```

Comment:
Might be worth adding a comment that spec only accepts JSON object param

Reply:
Yep comment added.